### PR TITLE
Add CI

### LINKED ISSUE #1: Setup CI
Taking what is there from https://github.com/varenc/homebrew-ffmpeg/issues/10:

```
FROM ubuntu:bionic
ARG DEBIAN_FRONTEND=noninteractive

RUN apt-get update \
    && apt-get install -y --no-install-recommends ca-certificates curl file g++ git locales make uuid-runtime python3-pip \
    && apt-get clean \
    && rm -rf /var/lib/apt/lists/* \
    && localedef -i en_US -f UTF-8 en_US.UTF-8 \
    && useradd -m -s /bin/bash linuxbrew \
    && echo 'linuxbrew ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers

USER linuxbrew
WORKDIR /home/linuxbrew
ENV PATH=/home/linuxbrew/.linuxbrew/bin:/home/linuxbrew/.linuxbrew/sbin:$PATH \
    SHELL=/bin/bash

RUN git clone https://github.com/Linuxbrew/brew.git /home/linuxbrew/.linuxbrew/Homebrew \
    && mkdir /home/linuxbrew/.linuxbrew/bin \
    && ln -s ../Homebrew/bin/brew /home/linuxbrew/.linuxbrew/bin/ \
    && brew config

RUN brew tap homebrew-ffmpeg/ffmpeg \
    && brew install homebrew-ffmpeg/ffmpeg/ffmpeg
```

Some example `.travis.yml` (untested):

```
anguage: ruby

matrix:
  include:
    - os: osx
      osx_image: xcode11.3
      rvm: 2.6
    - os: linux
      dist: xenial
      rvm: 2.6
  fast_finish: true
  allow_failures:
    - os: linux
      dist: xenial
      rvm: 2.6

env:
  global:
    - HOMEBREW_DEVELOPER=1
    - HOMEBREW_VERBOSE=1

before_install:
  - if [ "${TRAVIS_OS_NAME}" = "linux" ]; then
      export PATH="~/.linuxbrew/bin:~/.linuxbrew/sbin:$PATH";
      export HOME=/home/linuxbrew;
      export LINUXBREW=$HOME/.linuxbrew;
      export HOMEBREW_DEVELOPER=1
      export HOMEBREW_FORCE_VENDOR_RUBY=1
      export HOMEBREW_NO_AUTO_UPDATE=1
      sudo apt-get -qq update;
      sudo mkdir $HOME;
      sudo chown "$USER:" $HOME;
      mkdir $LINUXBREW;
      curl -L https://github.com/Linuxbrew/brew/tarball/master | tar xz -m --strip 1 -C $LINUXBREW;
    fi
  - brew update || brew update
  - if [ -f ".git/shallow" ]; then travis_retry git fetch --unshallow; fi
  - mkdir -p $(brew --repo)/Library/Taps/denji
  - ln -s $PWD $(brew --repo)/Library/Taps/denji/homebrew-nginx
  - cd $(brew --repo)/Library/Taps/denji/homebrew-nginx
  - export TRAVIS_BUILD_DIR="$(brew --repo)/Library/Taps/denji/homebrew-nginx"
  - ulimit -n 1024
  # Fix linux error: Incorrect file permissions (664)
  - chmod 0644 Formula/*.rb
  - umask 022

script:
  - brew test-bot --no-bottle

notifications:
  email:
    on_success: never
    on_failure: always
```

### PR DESCRIPTION
**Pull request by ddelange**
Since GH Actions is now in production and free for all public repos, there is now a [`brew test-bot`](https://github.com/Homebrew/homebrew-test-bot) command that will check syntax, installation etc.

Live Mac + Linux CI on my fork:
https://github.com/ddelange/homebrew-ffmpeg/pull/1

Closes #1

### CODE REVIEW

File: Formula/ffmpeg.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class Ffmpeg < Formula
   url "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.xz"
   version "4.3.1-with-options" # to distinguish from homebrew-core's ffmpeg
   sha256 "ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
-  revision 2
+  revision 3
```
Comment (retokromer) [inline Formula/ffmpeg.rb:7]: You could possibly also add
```rb
license "GPL-2.0-or-later"
```
just before.

Comment (ddelange) [reply Formula/ffmpeg.rb:8]: good one, actually just did it on homebrew-core too ^^
https://github.com/Homebrew/homebrew-core/pull/60410#issuecomment-683584725

Comment (retokromer) [reply Formula/ffmpeg.rb:8]: perfect, thanks a lot!

**Comment by retokromer** [comment]
@slhck Could you trigger Travis?

**Comment by ddelange** [comment]
@retokromer on my fork (https://github.com/ddelange/homebrew-ffmpeg/pull/1), CI still fails on ubuntu (downstream dependency perl fails to install somehow). I'm not sure how to fix that one.

On homebrew-core (https://github.com/Homebrew/homebrew-core/pull/60410), the `texinfo` dependency was removed altogether as it comes with vanilla MacOS, but I'm guessing that won't be the case on ubuntu.

Any help getting CI to pass on ubuntu would be awesome!

**Comment by slhck** [comment]
> @slhck Could you trigger Travis?

Do we still need Travis once this PR is implemented?
Seeing that homebrew-core also uses GH Actions, we might choose to do the same.

**Comment by retokromer** [comment]
@ddelange I haven’t checked previously your fork, sorry for the noise. I’ll give a check this evening (it’s morning here).

@slhck In my opinion, once it’s merged, we don’t need Travis anymore.

**Comment by retokromer** [comment]
@slhck Now Travis succeeded. If it’s fine with you, I’ll squash and merge.

@ddelange Thank you very much for your contributions here and on Homebrew!

**Comment by ddelange** [comment]
Yes, please do squash this 😅 

So ideally, `--skip-recursive-dependents` should not be present, as removing it will allow testing the full cycle. Since perl bottle seems to be broken on ubuntu test-bot (maybe I should create a follow-up issue to https://github.com/Homebrew/linuxbrew-core/issues/4808), I think running test-bot on homebrew-ffmpeg in ubuntu will keep [failing](https://github.com/ddelange/homebrew-ffmpeg/runs/1050581597#step:3:1422). On Mac all is good.

**Comment by slhck** [comment]
Please go ahead and merge. Thank you!

**Pull request merged**